src/styles/components/public-homepage.css

```
@@ -1,19 +1,28 @@
-  /* width: 100%; if using 100% margin-inline is redundant
-  border: lime 1px solid; */
-
-  padding: clamp(24px, 6dvh, 56px) var(--spacing-xl) var(--spacing-3xl);
+  width: 100%;
+  height: 100%;
 }
 
 .public-homepage__content {
+  position: relative;
+
+  display: flex;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)